websites/R/RTL+/metadata.json

```
@@ -17,7 +17,7 @@
 		"de": "RTL+ bietet Dir spannende Filme und Serien. Auch viele Shows und Dokus sind im Online-Stream zu finden! Deine Lieblingssendung jetzt online anschauen!"
 	},
 	"url": "www.tvnow.de",
-	"version": "1.0.1",
+	"version": "1.0.2",
```

Should be `1.1.0` because you added functionality
```suggestion
	"version": "1.1.0",
```